app/controllers/custom_styles_controller.rb

```diff
@@ -148,12 +154,7 @@ def update_themes
     call.on_failure do
       flash[:error] = call.message
     end
-
-    redirect_to action: :show
-  end
-
-  def show_local_breadcrumb
-    false
+    redirect_to custom_style_path
```

Comment:
Nitpick: add a blank line above this to improve readability and consistency.

Reply:
About the Rubocop error:

You can either ignore it, or move the theme selection into a private method:
```ruby
  def update_themes
    call = ::Design::UpdateDesignService
      .new(theme_from_params)
      .call

    call.on_success do
      flash[:notice] = I18n.t(:notice_successful_update)
    end

    call.on_failure do
      flash[:error] = call.message
    end

    redirect_to custom_style_path
  end

  private

  def theme_from_params
    OpenProject::CustomStyles::ColorThemes.themes.find { |t| t[:theme] == params[:theme] }
  end
```